sanitiser/_details.js

```diff
@@ -19,12 +19,15 @@ function sanitize( req, default_value ){
 
   if (params.details !== undefined) {
     var details = params.details;
-    
+
     if (typeof params.details === 'string') {
-      details = params.details === 'true';
```

Comment:
Is there any other code that does similar checking for "truthy"[1]? This giant conditional screams helper method to me. Its not a problem if this is the only place, but if it's not, we could do a lot to guarantee consistency in the API by making a helper method to do this sort of checking.

[1] I mean as we define truthy here, which includes 'y', 'yes', and whatever else we decide to include, not Javascripts concept of truthy